components/server/src/server.rs

```diff
@@ -231,6 +240,7 @@ struct TiKvServer<ER: RaftEngine> {
     background_worker: Worker,
     sst_worker: Option<Box<LazyWorker<String>>>,
     quota_limiter: Arc<QuotaLimiter>,
+    recovery_mode: bool,
```

Comment:
`recovery_mode` is not a clear name as TiKV can recover from many cases. I prefer `br_snapshot_recovery_mode`.

Reply:
And please add some comments to show what is snapshot backup/restore and how it works.